source/module_lr/potentials/pot_hxc_lrtd.cpp

```diff
@@ -36,18 +36,18 @@ namespace LR
             {
                 const std::set<std::string> lda_xc = { "lda", "pwlda" };
                 assert(lda_xc.count(this->xc_kernel_));
-                const int spinsize = (1 == nspin) ? 1 : 3;
-                std::vector<double> v2rho2(spinsize * nrxx);
+                const int n_component = (1 == nspin) ? 1 : 3;
```

Comment:
I think add a short annotation following this line will be good, because it is still not clear about what is "n_component". Because, yes I can realize it is the number of some "component" but then what? what kind of component?